cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/xcm_config.rs

```diff
@@ -250,7 +250,7 @@ impl pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmRouter = XcmRouter;
 	// We want to disallow users sending (arbitrary) XCMs from this chain.
-	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, ()>;
+	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
```

Comment:
IMO this can not be enabled on BH.

Reply:
Reverted in https://github.com/claravanstaden/polkadot-sdk/pull/4/commits/89746964578648edd2e9eebea83ad39b065d4d01.